samples/LocalEnvSecretStoreSample/Services/LocalEnvSecretStore.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Dapr.PluggableComponents.Components;
+using Dapr.PluggableComponents.Components.SecretStores;
+
+namespace LocalEnvSecretStoreSample.Services;
+
+internal sealed class LocalEnvSecretStore : ISecretStore
+{
+    private readonly ILogger<LocalEnvSecretStore> logger;
+
+    public LocalEnvSecretStore(ILogger<LocalEnvSecretStore> logger)
+    {
+        this.logger = logger;
+    }
+
+    #region ISecretStore Members
+
+    public Task<SecretStoreGetResponse> GetAsync(SecretStoreGetRequest request, CancellationToken cancellationToken = default)
+    {
+        this.logger.LogInformation("Get request for secret {key}", request.SecretName);
+
+        SecretStoreGetResponse? response = null;
+        string data = Environment.GetEnvironmentVariable(request.SecretName);
+        if (data == null)
+        {
+            data = "";
+        }
+        Dictionary<string, string> resp = new Dictionary<string, string>();
+        resp.Add(request.SecretName, data);
+        response = new SecretStoreGetResponse
```

Comment:
`var response = new SecretStoreGetResponse` allows you to eliminate the nullable `response` created above.